perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -131,6 +131,15 @@
 SECONDS_UNIT = 'seconds'
 MS_UNIT = 'milliseconds'
 
+MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
```

Comment:
Is it possible to also use these constants in the flags definition?

e.g., change:
flags.DEFINE_integer('mysql_svc_oltp_table_size', 100000,
                     'The number of rows of each table used in the oltp tests')
to
flags.DEFINE_integer(MYSQL_SVC_OLTP_TABLE_SIZE, 100000,
                     'The number of rows of each table used in the oltp tests')

to do that we probably need to move these string constants definitions up before the flags section.